.github/workflows/docker-latest.yml

```diff
@@ -0,0 +1,48 @@
+name: Publish Docker image for latest release
+
+on:
+  push:
+    tags:
+      - release
+
+env:
+  TEST_TAG: zeek:latest
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Checkout submodules
+        run: git submodule update --init --recursive
```

Comment:
You should be able to use `actions/checkout` with `submodules: recursive` option to do the submodule update ... worth a shot?